### ISSUE TITLE
should space character take label?

### ISSUE THREAD

**Issue by msobhan69** [MaybeShewill-CV/CRNN_Tensorflow#118]
In ASR task, silence is labeled for CTC loss. Isn't need for OCR?

**Comment by Mr-zihao** [comment]
yes, the last label is reversed for blank label.

**Comment by ryanwcyin** [comment]
The CTC part is implementing by using tf.nn.ctc_loss, in which the number of class = number of label +1 and and the largest value (num_classes - 1) is reserved for the blank label. For detail, read [
tf.nn.ctc_loss](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_loss)
